membership_policy.md

```diff
@@ -0,0 +1,125 @@
+# Community membership
+
+This document outlines the various responsibilities of contributor roles in KubeVirt.
+
+| Role  | Responsibilities | Requirements | Defined by
+| ----- | ---------------- | ------------ | ----------
+| Contributor | Adhere to the [KubeVirt code of conduct](https://github.com/kubevirt/community/blob/master/code-of-conduct.md) | Submit at least one pull request | Active GitHub account
+| Member | Active contributor in the community | * Active contributor<br>* Sponsored by 2 members<br>* Multiple contributions to the project | KubeVirt GitHub org member
+| Reviewer | Review contributions from others | * Member<br>* History of review and authorship in the project<br> * Sponsored by an approver | [OWNERS](https://github.com/kubevirt/kubevirt/blob/master/OWNERS_ALIASES) file reviewer entry |
+| Approver| Approve accepting contributions | * Reviewer<br>* Highly experienced<br> * Active reviewer & contributor to the project |  Sponsored by 2 approvers| [OWNERS](https://github.com/kubevirt/kubevirt/blob/master/OWNERS_ALIASES) file approver entry |
```

Comment:
There seems to be an additional column here. `Sponsored by 2 approvers` appears in the wrong one.